followthemoney/namespace.py

```diff
@@ -69,14 +77,19 @@ def apply(self, proxy):
             if prop.type != registry.entity:
                 continue
             for value in signed.pop(prop):
-                value = get_entity_id(value)
-                signed.add(prop, self.sign(value))
+                _value = get_entity_id(value)
+                signature = self.sign(_value)
+                if signature:
+                    signed.add(prop, signature)
+                else:
```

Comment:
signed.add handles None transparently, no need to bother with it here.

Reply:
:+1: